grpcclient/grpcclient.go

```diff
@@ -100,6 +103,9 @@ func (cfg *Config) DialOption(unaryClientInterceptors []grpc.UnaryClientIntercep
 	}
 	opts = append(opts, tlsOpts...)
 
+	unaryClientInterceptors = append(unaryClientInterceptors, cfg.GRPCMiddleware...)
```

Comment:
Could you document how the configured interceptors are ordered with respect to the ones passed to this method?